Cards/MadGraph5_aMCatNLO/TT/TTLL_MLL-10to50_amcatnlo-pythia8/TTLL_MLL-10to50_amcatnlo-pythia8.json

```diff
@@ -0,0 +1,23 @@
+{
+    "template": "NLO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "0",
+        "maxjetflavor": "5"
+    },
+    "model_params": "massW.dat",
+    "model_params_user": [
+        "set run_card ptj 20",
+        "set run_card mll_sf 10.0"
+    ],
+    "model_params_vars": {},
+    "fragment": ["Generator/ExternalLHEProducer.dat", "PartonShower/amcatnloFXFX-pythia8.dat"],
+    "fragment_user": [],
+    "fragment_vars": {
+        "processParameters": [
+            "'TimeShower:nPartonsInBorn = 2'",
```

Comment:
Ah I totally forgot about this.
@agrohsje confirm please : for plain amcatnlo samples that are not using FXFX, we should specify this right?

Reply:
Yes. It should be specified. I checked and the logic only works when there is a npNLO key word which we don't have in plan MC@NLO.